bin/commander

```diff
@@ -8,30 +8,84 @@ program :version, Commander::VERSION
 program :description, 'Commander utility program.'
 
 command :init do |c|
-  c.syntax = 'commander init <file>'
+  c.syntax = 'commander init [option] <file>'
   c.summary = 'Initialize a commander template'
   c.description = 'Initialize an empty <file> with a commander template,
     allowing very quick creation of commander executables.'
-  c.example 'Create a new file with a commander template.', 'commander init bin/my_executable'
-  c.action do |args, _options|
+  c.example 'Create a new classic style template file.',
+            'commander init bin/my_executable'
+  c.example 'Create a new modular style template file .',
+            'commander init --modular bin/my_executable'
+  c.option '-m', '--modular', 'Initialize a modular style template'
+  c.action do |args, options|
     file = args.shift || abort('file argument required.')
     name = ask 'Machine name of program: '
     description = ask 'Describe your program: '
     commands = ask_for_array 'List the commands you wish to create: '
-    begin
-      File.open(file, 'w') do |f|
-        f.write <<-"...".gsub!(/^ {10}/, '')
+    if options.modular
+      begin
+        File.open(file, 'w') do |f|
+          f.write <<-"...".gsub!(/^ {10}/, '')
```

Comment:
I think 10 here needs to be updated to 12 since this block is indented one level further.